### ISSUE TITLE
Tests for updatePlaylist and removeFromPlaylist functions

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/javascript-objects#34]
According to the instructions, the functions updatePlaylist and removeFromPlaylist are supposed to change the playlists sent to them. That is all working well.

However, as indicated by the instructions and the given solution, these functions are also supposed to return the updated playlists. The problem is that the test/objects-test.js file doesn't check for that. Thus, I was able to write this code and pass the tests without a problem.

function updatePlaylist (playlist, nameOfArtist, titleOfSong) {
  playlist[nameOfArtist] = titleOfSong;
  return;
}

function removeFromPlaylist (playlist, artistName) {
  delete playlist[artistName];
  return;
}
In short, all that's needed is for the tests to make sure that the functions return the updated playlists. Hope this helps!

Regards,

Steven

**Comment by hp77-creator** [comment]
Thanks a lot @Sdcrouse, I was returning the object and that is why was stuck at the lab for sometime.

**Comment by Sdcrouse** [comment]
@hp77-creator No problem! Glad I could help.
